ConfigService/src/main/java/com/revature/ConfigServlet.java

```diff
@@ -0,0 +1,147 @@
+package com.revature;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import org.kohsuke.github.GHEvent;
+import org.kohsuke.github.GHOrganization;
+import org.kohsuke.github.GHOrganization.Permission;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHUser;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class ConfigServlet extends HttpServlet {
+    final String ORG_NAME = "revaturelabs";
```

Comment:
Could we set these with a config file? That way it is easy for Revature to change without changing source code.